.vscode/settings.json

```diff
@@ -1,4 +1,4 @@
 {
-  "java.configuration.updateBuildConfiguration": "interactive",
+  "java.configuration.updateBuildConfiguration": "automatic",
```

Comment:
Generally, we don't need to commit settings.json (not everyone will be using vscode). If it requires specific configs, we can document them in the readme.